doc/modules/ensemble.rst

```diff
@@ -896,6 +895,19 @@ based on permutation of the features.
 
  * :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_regression.py`
 
+.. topic:: References
+
+  .. [F1999] Friedman, J.H. (2002). `Stochastic gradient boosting.
```

Comment:
```suggestion
  .. [F1999] Friedman, J.H. (1999). `Stochastic gradient boosting.
```

Reply:
The published paper https://doi.org/10.1016/S0167-9473(01)00065-2 is from 2002. As it is not open access, we give a different link (I guess).

Reply:
Ah I see. Can we update the reference to be consistent?

```suggestion
  .. [Friedman2002] Friedman, J.H. (2002). `Stochastic gradient boosting.
```